.github/workflows/labeler.yml

```
@@ -0,0 +1,135 @@
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      changed-ui: ${{ steps.set-ui-label.outputs.changed }}
+      changed-web: ${{ steps.set-web-label.outputs.changed }}
+      changed-root: ${{ steps.set-root-label.outputs.changed }}
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)